packages/js/src/ui/icons/Chat.tsx

```diff
@@ -2,12 +2,11 @@ import { JSX } from 'solid-js';
 
 export const Chat = (props?: JSX.HTMLAttributes<SVGSVGElement>) => {
   return (
-    <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 20 20" {...props}>
+    <svg width="10" height="10" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
Are we keeping width and height in icons or not? Currently some icons have it, other's dont.

Reply:
i removed all sizes